ecodse_funtime_alpha/data.py

```diff
@@ -59,18 +97,29 @@ def get_dataset(image_dir, labels_csv):
     return img_label_ds
 
 
-if __name__ == "__main__":
-    image_dir = '/project/def-bengioy/hrb/rainforest/fixed-train-jpg/'
-    labels_csv = '/project/def-bengioy/hrb/rainforest/train_v2.csv'
-    dataset = get_dataset(image_dir, labels_csv)
-
-    for n, sample in enumerate(dataset.take(4)):
-        image, label = sample[0], sample[1]
-        image = image.numpy()
-        plt.imshow(image)
-        plt.grid(False)
-        plt.xticks([])
-        plt.yticks([])
-        plt.title(label)
-        plt.show()
-        plt.close()
+def get_labels_distribution(labels_csv):
+    """
+    Returns a dict where the keys are the labels and the values are the number of occurences.
+
+    Parameters
+    ----------
+    labels_csv : str
+        Path to the csv file
+
+    Returns
+    -------
+        dict
+    """
+    # Read the csv
+    sample_labels = []
+    with open(labels_csv) as csvfile:
+        reader = csv.reader(csvfile, delimiter=',')
+        for i, row in enumerate(reader):
+            if i < 1:  # Skip first row as it is column names
+                continue
```

Comment:
while this is a correct method of skip the first row, you used next(read) earlier (line 67). For sake of consistency, it's probably best to use the same thing here